build/BuildFunctions.psm1

```diff
@@ -263,3 +263,18 @@ function Get-TypeScriptTasks() {
     }
     $dirs.Name | Write-Output
 }
+
+function Update-AppveyorBuildVersion () {
+    if (!$env:APPVEYOR) {
+        Write-Error "Only avalable when running in Appveyor."
```

Comment:
Will write error throw or will this continue?

Reply:
It can be configured to throw, but in general will just write an error to the error pipeline.
Adding a return statement to short-circuit the body function when this is false.